components/token-operation/HotWalletBroadcastTokenOperation.tsx

```diff
@@ -73,7 +74,7 @@ export default function HotWalletBroadcastTokenOperation({
       }
 
       const commitTxId = await account.signAndBroadcastTx([
-        { amount: "0.1", address: P2SHAddress.toString() },
+        { amount: "0.3", address: P2SHAddress.toString() },
```

Comment:
It would be better to get rid of magic number with `MINIMUM_UTXO_AMOUNT` constant variable.

Reply:
d7599ab47a114daca641dbfc67269fe23707dced